### PR TITLE
Implement list Structure design

### PR DESCRIPTION
**Pull request by bienvenuushindi**
- Set up a new project
- Create an index.html file with HTML markup .
- Create an empty To Do List placeholder .
- Create an index.js file and set an array of some simple to do tasks (array of objects).
- Each task object should contain three keys:     
  - description
  - [string], completed [bool].
  - index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+import Task from './modules/Task.js';
+import { list } from './modules/elements.js';
 import './style.css';
+import listItem from './modules/list-item';
 
-function component() {
-  const element = document.createElement('div');
+const init = () => {
+  const task1 = new Task('wash the dishes');
+  const task2 = new Task('complete to Do list project');
```
Comment (Kingstalux) [inline src/index.js:8]: - [x] Good job so far 👍 but you are to create an array of your tasks and not a class. Your tasks array should contain task objects which have a description, completed, and index keys. Kindly address this issue 💯 
- Requirements:
Screenshot 2022-07-07 090348

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+  task1.appendToList();
+  task2.appendToList();
+  list.innerHTML += Task.list();
```
Comment (Kingstalux) [inline src/index.js:11]: - [x] You are to write a function that takes the tasks array as an argument and populate your HTML 👍 Kindly rectify this issue. Take note that you do not need to implement any extra functionality but kudos for currently implementing the add tasks functionality 🔥 
- Requirements:
Screenshot 2022-07-07 090553

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @bienvenuushindi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
175083599-879bf673-fb93-406f-939c-de7e586bf6d9


## Highlights 👏 
✅ Good UI design 🔥 
✅ Good commit history 👍 
✅ Linters are passing 👏 


## Required Changes ♻️
_Check the comments under the review._
- [x] KIndly edit your README file with your information and information about your project like how a user can set up and run it locally on their machine 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (bienvenuushindi) [reply src/index.js:8]: @Kingstalux Thanks For the feedback I really appreciate :+1:  I Think I created The Array of tasks [Please verify this line of my code line 4](https://github.com/bienvenuushindi/to-do-list/blob/list-structure/src/modules/Task.js#L4) 
Concerning the highlighted block of code  I just created 2 tasks then I push them to my array using the [appendToList line 11-13](https://github.com/bienvenuushindi/to-do-list/blob/list-structure/src/modules/Task.js#L11-L13) method

File: src/index.js (hunk 2 above)
Comment (bienvenuushindi) [reply src/index.js:11]: @Kingstalux  [Here is my function Line 16-18 ](https://github.com/bienvenuushindi/to-do-list/blob/list-structure/src/modules/Task.js#L16-L18) 

I think I implemented everything as  requested  I just did it implicitly

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @bienvenuushindi 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**